flang/lib/Optimizer/Builder/Runtime/Command.cpp

```diff
@@ -13,6 +13,18 @@
 
 using namespace Fortran::runtime;
 
+// Cetain runtime intrinsics should only be ran when select parameters of the
+// intrisic are supplied. In certain cases one of these parameter may not be
+// given, however we wish to run the intrinsic due to another required parameter
+// being supplied. In this case we assign the missing parameter to have an
+// "absent" value. This typically happens in IntrinsicCall.cpp. For this reason
+// we need extra indirection with `isAbsent` for testing whether a given
```

Comment:
Can you rephrase to not us the pronoun "we" in this comment?

Reply:
Done.